### PR TITLE
Practice opencv

### PR DESCRIPTION
**Pull request by ZheleztsovaIrina**

### CODE REVIEW

File: modules/0_git/test/main.cpp [hunk 1]
```diff
@@ -7,7 +7,7 @@
 TEST(git, say_hello) {
     myspace::A a;
     EXPECT_EQ(myspace::func(a), "Hello, Nizhny!");
-    EXPECT_EQ(func(a), "Hello, World!");
+    EXPECT_EQ(func(a), "Hello, Nizhny!");
```
Comment (dkurt) [inline modules/0_git/test/main.cpp:10]: Please do
```
git checkout master
git checkout -b do_opencv
```
for every new module

Comment (dkurt) [reply modules/0_git/test/main.cpp:10]: Otherwise you have the changes from previous tasks

File: modules/1_opencv/src/opencv_coins.cpp [hunk 2]
```diff
@@ -11,4 +14,25 @@ unsigned countCoins(const Mat& img) {
...
+    minMaxLoc(thresh, &dist_transform);
+    threshold(thresh,thresh, 0.7*dist_transform,255.0, THRESH_BINARY);
+    thresh.convertTo(thresh, CV_8U);
+    std::vector<std::vector<cv::Point> > contours;
+    cv::findContours(thresh, contours, RETR_EXTERNAL, CHAIN_APPROX_SIMPLE);
+    int ncomp = contours.size();
+    int sum = 0;
+    double radius;
+    for (int i = 0; i < ncomp; ++i) {
+        radius = contourArea(contours[i]) / (2 * 3.14);
```
Comment (dkurt) [inline modules/1_opencv/src/opencv_coins.cpp:29]: Is that really radius? Can you please remind circle radius formula?

Comment (ZheleztsovaIrina) [reply modules/1_opencv/src/opencv_coins.cpp:29]: s=pi*r^2

Comment (dkurt) [reply modules/1_opencv/src/opencv_coins.cpp:29]: So why we divide by 2?

Comment (ZheleztsovaIrina) [reply modules/1_opencv/src/opencv_coins.cpp:29]: I fixed this mistake, but the tests failed

Comment (dkurt) [reply modules/1_opencv/src/opencv_coins.cpp:29]: Please check https://github.com/dkurt/openvino_practice/pull/85/files - the changes are the same

File: modules/1_opencv/src/opencv_coins.cpp [hunk 3]
```diff
@@ -11,4 +14,25 @@ unsigned countCoins(const Mat& img) {
 
     // TODO: implement an algorithm from https://docs.opencv.org/master/d3/db4/tutorial_py_watershed.html
     CV_Error(Error::StsNotImplemented, "countCoins");
```
Comment (dkurt) [inline modules/1_opencv/src/opencv_coins.cpp:16]: You need to remove this assertion

Comment (ZheleztsovaIrina) [reply modules/1_opencv/src/opencv_coins.cpp:16]: I did it

Comment (dkurt) [reply modules/1_opencv/src/opencv_coins.cpp:16]: Where?

**Comment by dkurt** [comment]
see https://github.com/dkurt/openvino_practice/pull/94

File: modules/1_opencv/src/opencv_coins.cpp (hunk 3 above)
Comment (ZheleztsovaIrina) [reply modules/1_opencv/src/opencv_coins.cpp:16]: It was the answer to 
Please check https://github.com/dkurt/openvino_practice/pull/85/files - the changes are the same

**Pull request closed without merging**